ibmdbpy/ae/result_builder.py

```diff
@@ -39,8 +39,11 @@ def build_result(output_table, merge_output, db, df, output_signature, table_nam
         df = idadf.as_dataframe()
         return df
     if output_table is None and merge_output is True:
-        output_table ="pyida_table"
-        create_string = "create table " + output_table + "_temp as "
+        output_table_prefix = "pyida"
```

Comment:
It looks as if you didn't use the variable `output_table_prefix` . So you can remove this statement.